users/forms.py

```diff
@@ -0,0 +1,5 @@
+from django import forms
+
+
+class ProfilePhotoForm(forms.Form):
```

Comment:
Why do you need this class? You do not use it.

Reply:
done